concepts/05 UI Components/DataGrid/70 Custom Keyboard Navigation/15 Override and Create Keys.md

```diff
@@ -0,0 +1,105 @@
+The [onKeyDown](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onKeyDown) event handler allows you to track keys used while the UI component is in focus. You can use it to override the default keyboard support shortcuts, implement your own shortcuts, or extend existing ones.
+
+The following example shows how to override the **Space** key to switch a cell in cell/batch mode to the editing state instead of the current row being selected:
```

Comment:
```suggestion
The following example shows how to override the **Space Bar** keystroke so it switches a cell in cell/batch mode to the editing state instead of a current row select operation:
```